ga4gh/exceptions.py

```diff
@@ -90,7 +90,14 @@ def toProtocolElement(self):
 #####################################################################
 
 
-class BadRequestException(BaseServerException):
+class RuntimeException(BaseServerException):
```

Comment:
We should move the httpStatus stuff into `RuntimeException` from `BaseServerException` I think. The DataExceptions occur when the server is starting up, and result in something being communicated back to the user via stderr or logs. The idea of HttpStatus doesn't mean very much here. RuntimeExceptions occur when we are serving HTTP requests, and they need all of the machinery for http status and error codes. I guess we should keep `message` in `BaseServerException` so that we have uniform interface for creating error messages.